sqeleton/databases/snowflake.py

```diff
@@ -164,25 +164,43 @@ def __init__(self, *, schema: str, **kw):
         assert '"' not in schema, "Schema name should not contain quotes!"
         # If a private key is used, read it from the specified path and pass it as "private_key" to the connector.
         if "key" in kw:
```

Comment:
I believe this would need to be 
```suggestion
        if "key" in kw or "key_path" in kw:
```

Reply:
in the case where only "key_path" is provided

Reply:
good catch